docs/src/pages/components/data-grid/accessibility/accessibility.md

```diff
@@ -71,6 +71,7 @@ Use the arrow keys to move the focus.
 |                                             <kbd class="key">CTRL</kbd> + <kbd class="key">A</kbd> | Select all rows                                   |
 |                                             <kbd class="key">CTRL</kbd> + <kbd class="key">C</kbd> | Copy the currently selected row                   |
```

Comment:
```suggestion
|                                             <kbd class="key">CTRL</kbd> + <kbd class="key">C</kbd> | Copy the currently selected row(s)                |
```

Reply:
This seems to be describing Windows modifier keys.

Reply:
I can log a separate issue and possibly a PR for docs later to consider Mac modifier keys.